target/linux/apm821xx/image/Makefile

```diff
@@ -224,7 +228,8 @@ endif
 ifeq ($(SUBTARGET),sata)
 
 define Device/wd_mybooklive
-  DEVICE_TITLE := Western Digital My Book Live Series (Single + Duo)
+  DEVICE_VENDOR := Western Digital
+  DEVICE_MODEL := My Book Live Series (Single + Duo)
```

Comment:
(Note: The MBL used to be two models, since they both really had very distinct boards. I'm not sure if unifying them was a good idea)

Reply:
If you like a change, please make a proposal. Thanks!